server/zoom/jwt.go

```diff
@@ -54,6 +54,16 @@ func (c *JWTClient) GetUser(user *model.User) (*User, *AuthError) {
 	return &zoomUser, nil
 }
 
+func (c *JWTClient) CreateMeeting(userEmail string) (*Meeting, error) {
+	var ret Meeting
+	meetingRequest := CreateMeetingRequest{
+		Topic: "Meeting created on Mattermost",
```

Comment:
Can we let the user specify a meeting topic via `/zoom start Let's have some fun!`

Reply:
That can be done, but I think it would be a great HW ticket, if you are willing to write it down.